Cargo.toml

```diff
@@ -5,6 +5,7 @@ members = [
     'node',
     'pallets/*',
     'runtime',
+    'supra-cli',
```

Comment:
Do we need a separate project for this? Different projects are usually created for code that is either shareable or collectively serves a different purpose.
Here, we already have node project that takes care of handling command line arguments. Having our code there, would make it easier to manage.

Reply:
Having a detached crate handling the CLIs would be best. 

The reason is because, we would still be having more CLIs in the future. With that, we can have a place to add more and just call it inside `node`. Hence, more clarity.